ios-base/DroidKaigi 2020/Session/ViewModels/SessionViewModel.swift

```diff
@@ -0,0 +1,47 @@
+import ios_combined
+import RxSwift
+import RxCocoa
+
+final class SessionViewModel {
+
+    private let disposeBag = DisposeBag()
+
+    // input
+    let viewDidLoad = PublishRelay<Void>()
+    let toggleEmbddedView = PublishRelay<Void>()
```

Comment:
I recommend to make `~ Relay` as `private` to prevent unexpected events or subscribing.
If you want to use it from outside, you may create wrapper property or function like this.

```swift
private let viewDidLoadRelay = PublishRelay<Void>()

// to fire event
func viewDidLoad() {
    viewDidLoadRelay.accept(())
}

// to subscribe
var viewDidLoad: Observable<Void> {
    viewDidLoadRelay.asObservable
}
```

Reply:
Certainly, I think this looks better than the current one. I will fix it. 
Thanks! 👍